timemachine/ff/handlers/deserialize.py

```diff
@@ -5,12 +5,13 @@
 from timemachine.ff.handlers.suffix import _SUFFIX
 
 
-def deserialize_handlers(obj):
+def deserialize_handlers(obj, protein_ff_name=constants.DEFAULT_PROTEIN_FF, water_ff_name=constants.DEFAULT_WATER_FF):
```

Comment:
You should make it clear that these are the ffs that this will fall back to if they are not specified in the object being deserialized.

Reply:
Sorry, I don't understand how this code works well enough to be confident in the most recent change https://github.com/proteneer/timemachine/pull/989/commits/171c17ce29f30f26a6e839046fe49c68eaa13055 or write an accurate description of this fallback logic -- will revert for now, and possibly revisit.

Reply:
For posterity.

When a forcefield (smirnoff_*.py) file is deserialized, this code will look for the relevant tag protein/water in the dictionary object. If the tag is not there, it will fall back to the values specified by the defaults.

Currently you can't control what the FF should default to, besides modifying the file format or the Forcefield object fields. The change you had reverted would have allowed users to specify a default of their own, though if the file being deserialized had defined a water/protein ff already it would have done nothing.

Reply:
It seems like we should prefer `Forcefield.serialize` or modify `serialize_handlers` to take a ff rather than handlers + other parameters, as the Forcefield is really what users care about. Is a bit confusing as it stands.

Reply:
That makes sense -- thanks for explaining!

Sounds like a useful change to be picked up in a separate PR, but want to do this carefully to avoid introducing new possibilities for mismatches.